src/style.css

```
@@ -1 +1,13 @@
 /* styles go here */
+.form-fieldset {
+  margin-bottom: 20px;
+}
+
+.form-fieldset :last-child {
+  margin-bottom: 0;
+}
+
+.form-field {
+  display: block;
```

You use this class only on div's, which are already have display: block by default, remove this redundant property
```suggestion
.form-field {
```